Botan3: Missing include (botan/dl_group.h)

## Overview
Building KeepassXC fails with Botan3.

## Steps to Reproduce
1. Install botan3
2. Clone develop
3. Try building

## Expected Behavior
Build should succeed.

## Actual Behavior
Build fails with error:

```
In constructor ‘FdoSecrets::DhIetf1024Sha256Aes128CbcPkcs7::DhIetf1024Sha256Aes128CbcPkcs7(const QByteArray&)’:
keepassxc/src/fdosecrets/objects/SessionCipher.cpp:43:113: error: invalid use of incomplete type ‘class Botan::DL_Group’
   43 |             m_privateKey.reset(new Botan::DH_PrivateKey(*randomGen()->getRng(), Botan::DL_Group("modp/ietf/1024")));
      |                                                                                                                 ^
In file included from keepassxc/src/fdosecrets/objects/SessionCipher.cpp:26:
/usr/include/botan-3/botan/dh.h:17:7: note: forward declaration of ‘class Botan::DL_Group’
   17 | class DL_Group;
      |       ^~~~~~~~
```

## Context

Operating System: Linux
Desktop Env: i3
Windowing System: X11
Compiler: GCC 13.1.1
Botan Version: 3.0.0

--- a/src/fdosecrets/objects/SessionCipher.cpp
+++ b/src/fdosecrets/objects/SessionCipher.cpp
@@ -26,6 +26,7 @@
 #include <botan/dh.h>
 
 #ifdef WITH_XC_BOTAN3
+#include <botan/dl_group.h>
 #include <botan/pubkey.h>
 #else
 #include <botan/pk_ops.h>

--- a/src/gui/styles/base/BaseStyle.cpp
+++ b/src/gui/styles/base/BaseStyle.cpp
@@ -1936,7 +1936,7 @@ void BaseStyle::drawPrimitive(PrimitiveElement elem,
         auto fropt = qstyleoption_cast<const QStyleOptionFocusRect*>(option);
         if (!fropt)
             break;
-        //### check for d->alt_down
+        // ### check for d->alt_down
         if (!(fropt->state & State_KeyboardFocusChange))
             return;
         if (fropt->state & State_Item) {

